Users/Install.rst

```diff
@@ -83,10 +83,82 @@ To install coala only (without any bears), you can do:
     **Ubuntu Users** - Do not use ``apt-get install coala``
     for installing coala as that is different software.
 
+Installing inside a pipenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pipenv is the most user friendly method of managing virtual environment and
+packages.
+You can look at it as a mix of ``pip`` and ``virtualenv``. So you don't have
+to manage them seperately for your projects.
+
+First, assuming you have installed python and pip earlier from
+this guide we'll now install ``pipenv`` by running the following command.
+
+::
+
+    $ pip install pipenv
+
+Now we create a python 3 environment for our project, move to your project
+directory and run the following command.
+
+::
+
+    $ pipenv --three
+
+The above command will create virtual environment isolated from your system
+global installation.
+
+    On **Unix based** systems (OS X and Linux), you can find the user base
+    binary directory for your pipenv by running
+
+    ::
+
+        $ python -m site --user-base
+
+    Result of the above command will be something similar to
+    ``/home/<Username>/.local``.
+    And on **Windows**, run the following command
+
+    ::
+
+        $ py -m site --user-site
+
+    Result of the above command will be something similar to
+    ``C:\Users\<Username>\AppData\Roaming\Python36\site-packages``.
+    You'll have to add this result after replacing ``site-packages``
+    with ``Scripts`` i.e.
+    ``C:\Users\<Username>\AppData\Roaming\Python36\Scripts``
+    to PATH from Control Panel. This is only when you don't have this PATH
```

Comment:
This entire block is a little hard to follow. Something similar to this might be better:

> Replace the occurrence of `site-packages` in the result of the command above with `Scripts` to get something like `C:\Users\<Username>\AppData\Roaming\Python36\Scripts`. Add this value to your `PATH` environment variable, which you can find in the Control Panel. Only follow this step if this value isn't already set in your `PATH`.

Reply:
Let me rephrase that, your version seems better.